### PR TITLE
IA-3979 Update azure_vm_init_script.sh

### PR DESCRIPTION
**Pull request by erdalcosgun**
Update the azure vm script for kernel deletions and update rbase.

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: http/src/main/resources/init-resources/azure_vm_init_script.sh [hunk 1]
```diff
@@ -62,6 +64,26 @@ RELAY_CONNECTIONSTRING="Endpoint=sb://${RELAY_NAME}.servicebus.windows.net/;Shar
...
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove pysparkkernel  
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove spark-3-python 
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove julia-1.6
+
+echo "Y"| /anaconda/envs/py38_default/bin/pip3 install ipykernel
+
+echo "Y"| /anaconda/envs/py38_default/bin/python3 -m ipykernel install
```
Comment (rtitle) [inline http/src/main/resources/init-resources/azure_vm_init_script.sh:85]: Ah - is this the key thing that fixes the earlier issue with the phantom `Python 3` kernel?

File: http/src/main/resources/init-resources/azure_vm_init_script.sh [hunk 2]
```diff
@@ -62,6 +64,26 @@ RELAY_CONNECTIONSTRING="Endpoint=sb://${RELAY_NAME}.servicebus.windows.net/;Shar
...
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove sparkkernel
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove sparkrkernel
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove pysparkkernel  
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove spark-3-python 
+
+echo "Y"| /anaconda/bin/jupyter kernelspec remove julia-1.6
```
Comment (rtitle) [inline http/src/main/resources/init-resources/azure_vm_init_script.sh]: It makes me 😢 to remove Julia -- should we keep it?

(Makes sense to remove the Spark ones)

**Review by rtitle — COMMENTED** [review]
I didn't test but LGTM. Wondering if we should keep Julia :)

File: http/src/main/resources/init-resources/azure_vm_init_script.sh (hunk 1 above)
Comment (erdalcosgun) [reply http/src/main/resources/init-resources/azure_vm_init_script.sh:85]: > Ah - is this the key thing that fixes the earlier issue with the phantom `Python 3` kernel?

Yes :)

File: http/src/main/resources/init-resources/azure_vm_init_script.sh (hunk 2 above)
Comment (erdalcosgun) [reply http/src/main/resources/init-resources/azure_vm_init_script.sh]: > It makes me 😢 to remove Julia -- should we keep it?
> 
> (Makes sense to remove the Spark ones)

Julia is still on the VM but not in Kernel list. I updated the PR and keep Julia kernel.

**Review by rtitle — APPROVED** [review]
Thanks!

**Comment by rtitle** [comment]
Tested on a BEE! Will merge when green.

**Comment by rtitle** [comment]
Tested manually, admin merging

**Pull request merged**